lib/plugins/aws/package/compile/functions.js

```diff
@@ -147,7 +148,7 @@ class AwsCompileFunctions {
 
   async addFileToHash(filePath, hash) {
     const lambdaHashingVersion = this.serverless.service.provider.lambdaHashingVersion;
-    if (lambdaHashingVersion) {
+    if (lambdaHashingVersion === '20201221') {
```

Comment:
Nit: Conceptually we should rather use `>=` comparison (although it's highly unlikely we will introduce any other mode in a future)